frontend/src/components/ActionMenuItems.ts

```diff
@@ -0,0 +1,20 @@
+import type { ActionMenuItem } from "~/components/ActionsMenu.vue";
+import PDFIcon from "~/components/icons/PDFIcon.vue";
+import MaterialSymbolsPrint from "~icons/material-symbols/print";
+
+export const printActionMenuItem: ActionMenuItem = {
```

Comment:
I am not sure if those constants belong here. They are rather constants than components. The folder of components should remain mainly for vue components that are reused in the project and potentially have unit tests that test their functionality. For reused constants, I'd go for another location.

Reply:
Right, I just didn't know where else to put them, any suggestions? :)

Reply:
If I were you I'd still keep them within their original components/pages.